subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -512,12 +536,11 @@ static int __wifi_args_to_params(size_t argc, char *argv[],
 
 			if (!found) {
 				print(context.sh, SHELL_ERROR,
-				      "Invalid channel: %ld, checked bands: %s\n",
-				       channel,
-				       bands_str);
+					"Invalid channel: %ld, checked bands: %s\n",
+					channel,
+					bands_str);
```

Comment:
Are these changes necessary? If so, maybe they should be part of a separate commit?